frog/imports/client/Wiki/index.js

```diff
@@ -110,7 +111,8 @@ class WikiComp extends Component<WikiCompPropsT, WikiCompStateT> {
       deletedPageModalOpen: false,
       currentDeletedPageId: null,
       currentDeletedPageTitle: null,
-      rightSideCurrentPageObj: null
+      rightSideCurrentPageObj: null,
+      user: Meteor.user()
```

Comment:
why would you put this in state? Why can't you just query Meteor.user() when you need it? It might make sense if you were actually listening to the Meteor.user and updating state when it changes, but you don't (and we don't expect it to change, except if we allowed renaming in the future, as discussed on Slack)

Reply:
On the server, this will fetch the record from the database. To improve the latency since it is being called multiple times, we could save the returned record to a variable instead of re-calling `Meteor.user()`.

Reply:
That's not how Meteor works - Meteor uses something called Minimongo, which subscribes to certain collections and always keeps them in sync locally. The only time a server call is made is when you make a new subscription (with withTracker) or when you call a Meteor method. Please revert this.

Reply:
Reverted the changes